examples/next/pages/ui/components/in-app-messaging/override-component/index.page.tsx

```diff
@@ -24,6 +24,7 @@ const CustomBannerMessage = (props) => {
       padding="xl"
       backgroundColor="teal.20"
       right="xl"
+      testId="custom-banner"
```

Comment:
Added testId as we just want to test that this custom component is shown as the banner. Figured it is easier to refer to this testId in tests than testing for styling.